### PR TITLE
chore(gh actions): upgrade to newer version

### PR DESCRIPTION
**Pull request by eyenx**
Updates some GH Actions stuff to newer Releases.

### CODE REVIEW

File: hack/deploy/init.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ source hack/sh/rc.sh
 
 readonly infra_ns=$ADFINIS_CHARTS_INFRA_NS
 readonly argocd_version=$ADFINIS_CHARTS_ARGOCD_VERSION
-readonly prometheus_operator_version=$ADFINIS_CHARTS_PROMETHEUS_OPERATOR_VERSION
+readonly kube_prometheus_stack_version=$ADFINIS_CHARTS_PROMETHEUS_OPERATOR_VERSION
```
Comment (hairmare) [inline hack/deploy/init.sh:9]:
```suggestion
readonly kube_prometheus_stack_version=$ADFINIS_CHARTS_KUBE_PROMETHEUS_STACK_VERISON
```

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -35,21 +35,21 @@ jobs:
 
       - name: Run chart-testing (lint)
         id: lint
-        uses: helm/chart-testing-action@v1.0.0
+        uses: helm/chart-testing-action@v2.0.1
```
Comment (hairmare) [inline .github/workflows/test.yaml:38]: v2 will only install `ct` and not run it: [Upgrading from v1.x.x](https://github.com/helm/chart-testing-action#upgrading-from-v1xx)

You might want to take a look at the [Example Workflow](https://github.com/helm/chart-testing-action#example-workflow) to see what else needs changing for v2 compat.

**Review by hairmare — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yaml (hunk 2 above)
Comment (eyenx) [reply .github/workflows/test.yaml]: 6509809

Comment (eyenx) [reply .github/workflows/test.yaml]: Thanks!

**Review by hairmare — APPROVED** [review]

**Comment by eyenx** [comment]
:fireworks:

**Comment by hairmare** [comment]
Thanks!

**Comment by eyenx** [comment]
This PR removed the `linting` functionality! 

@tongpu @hairmare 

Is now fixed in https://github.com/adfinis-sygroup/helm-charts/pull/144

**Pull request merged**